docs/guide/execute-transaction.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: splash
+---
+
+# Execute Transaction
+
+`executeTransaction` is a high level function which can be used to perform transactions on Algorand Network. It supports every transaction (atomic or single) which is possible in network. Ex: Deploy ASA/SSC, Opt-In, Transfers, Delete, Destroy etc. `executeTransaction` takes `ExecParams` or `ExecParams[]` as parameter.
+If you pass an array of `ExecParams`, it will be considered as `atomic transaction`.
+In below sections we will demonstrate how to pass these parameters.
+
+Note: For each parameter `type` and `sign` fields are mandatory.
+  - `type`: type of transaction
+  - `sign`: sign type
+
+Examples of parameter(`ExecParams`):
+# Transfer Algo using secret key
```

Comment:
```suggestion
### Transfer Algo using secret key
```

Reply:
First level header should be use only for title.
Let's update the headers below as well

Reply:
ok